### ISSUE TITLE
netcdf4 installs weird versions of NetCDF utilities and have issue with libjpeg

### ISSUE THREAD

**Issue by silentquasar** [Anaconda-Platform/support#98]
Hi,

System:
CentOS 7 64-bit
Anaconda 3 - 4.3.0 x64
Installed netcdf4: `conda install netcdf4`

This seems to include some weird versions of the NetCDF utilities (ncgen, ncdump, etc.). 

First of all, they complain about a missing libjpeg.so.8. If I install that: `conda install jpeg=8d`, then they run but don't match the current versions of the utilities at all--they don't give a NetCDF version with the -V command, just an HDF version: `ncgen, HDF Version 4.2 Release 11, February 5, 2015`. What gives? Are these utilities from the HDF library instead of the NetCDF library?

I can get around the issue by installing NetCDF in the OS with `yum` and then forcing my application to use `/usr/bin/ncgen`, but that shouldn't be necessary. The version string for ncgen should be something like `netcdf library version 4.3.3.1 of Dec 10 2015 16:44:18 $`

TL;DR: Please include the proper NetCDF utilities with the NetCDF package, or inhibit these weird ones from showing up.

**Comment by silentquasar** [comment]
The offending utilities appear to be in the hdf4 package.

**Comment by silentquasar** [comment]
I guess the current state is not so much "broken" as it is not what I expected. Does anyone expect to use the ncgen and ncdump utilties from the hdf4 package, rather than those from the NetCDF package?

**Comment by silentquasar** [comment]
This has been addressed in conda-forge:
https://github.com/conda-forge/hdf4-feedstock/issues/5

**Comment by silentquasar** [comment]
This should have been in https://github.com/ContinuumIO/anaconda-issues/issues/1346. I'll add my notes to that issue.
